<launch>
	<arg name = "orientation_frame" default = "local" />
	<arg name = "position_frame" default = "global" />
	<arg name = "control_offset" default = "true" />
	
	<param name="goalset/orientationFrame" value="$(arg orientation_frame)" />
	<param name="goalset/positionFrame" value="$(arg position_frame)" />
	<param name="goalset/controlOffset" value="$(arg control_offset)" />
	
	<node name = "set_goal" pkg = "usarsim_set_goal" type = "set_goal" />
</launch>
